packages/kbn-interpreter/src/common/interpreter/interpret.js

```diff
@@ -103,7 +102,7 @@ export function interpretProvider(config) {
     }
 
     // Validate the function output against the type definition's validate function
-    const type = typesRegistry.get(fnDef.type);
+    const type = config.types[fnDef.type];
```

Comment:
nit: you're destructuring the types off the config object above, and even re-assigning the values to `handlers`. This is the only place you're using values on the `config` object, but you could use `handlers` here instead.

Reply:
won't handlers have additional stuff ? (config.handlers)